docs/variant_annotation.md

```diff
@@ -138,3 +144,34 @@ variants.
 <!-- TODO(bashir): Add some interpretation of these variants, e.g., with
 reference to sickle-cell and thalassemia. This needs to be done using other
 fields to extract phenotype information from other databases. -->
+
+### Joining with other databases
+
+You can also join with other databases that are available in BigQuery.
+For instance, there are several pathway databases that are available publicly in
+BigQuery. Here is a sample query to extract high impact variants that are in
+the double-strand break repair pathway as defined by the
+[GO (Gene Ontology) biological process (GO:0006302)](http://amigo.geneontology.org/amigo/term/GO:0006302).
+
+```
+#standardSQL
+SELECT
+  reference_name, start_position, reference_bases, ALT.alt, CSQ.*
+FROM
+  vcf_imports_external.gnomad_genomes_chr_hg19 AS T, T.alternate_bases AS ALT, ALT.CSQ AS CSQ
+WHERE
+  CSQ.SYMBOL IN (SELECT DB_Object_Symbol
+                 FROM `isb-cgc.genome_reference.GO_Annotations`
+                 WHERE GO_ID = 'GO:0006302')
+  AND CSQ.IMPACT = "HIGH"
+ORDER BY start_position
+```
+
+![A sample query joining annotations with GO database](images/gnomAD_GO_0006302.png)
+
+Other pathway databases are also available in BigQuery including
+[Reactome](https://reactome.org/)
+([table](https://bigquery.cloud.google.com/table/isb-cgc:genome_reference.Ensembl2Reactome?tab=details))
+and [WikiPathways](https://www.wikipathways.org)
+([table](https://bigquery.cloud.google.com/table/isb-cgc:QotM.WikiPathways_20170425_Annotated?tab=details)).
```

Comment:
I think it is better if we add some explicit acknowledgement/reference to [ISB-CGC](https://isb-cgc.appspot.com/) as I think these are all created and published as part of that effort.